src/Sulu/Bundle/CategoryBundle/Tests/Functional/Controller/CategoryControllerTest.php

```diff
@@ -214,20 +214,32 @@ public function testCGet()
 
         $this->assertHttpStatusCode(200, $client->getResponse());
 
-        $response = json_decode($client->getResponse()->getContent());
+        $response = json_decode($client->getResponse()->getContent(), true);
+
+        $categories = $response['_embedded']['categories'];
+
+        // results are unordered, and order is inconsistent between database drivers
+        foreach ($categories as $category) {
```

Comment:
Why not using something like `usort`? The logic you've added here gets quite complicated once a few more categories are added.